### PR TITLE
PMM-11023 Move OLM from dbaas-controller to pmm

### PR DESCRIPTION
**Pull request by percona-csalguero**
PMM-11023 Move OLM from dbaas-controller to pmm.

- Moved code from dbaas-controller to pmm-managed.
- Deprecated the use of kubectl in favor of Kubernetes go-client.
- Fixed timeout when generating Grafana API keys.
- Fixed error because of an empy env-var to set the pmm client image.

### CODE REVIEW

File: managed/services/management/dbaas/kubernetes_server.go [hunk 1]
```diff
@@ -270,37 +272,8 @@ func replaceAWSAuthIfPresent(kubeconfig string, keyID, key string) (string, erro
 	return string(c), err
 }
 
-// TODO: @Carlos do we still need it?
```
Comment (gen1us2k) [inline managed/services/management/dbaas/kubernetes_server.go:273]: No. We do not need it I think

File: managed/services/management/dbaas/kubernetes_server.go [hunk 2]
```diff
@@ -341,159 +316,129 @@ func (k kubernetesServer) RegisterKubernetesCluster(ctx context.Context, req *db
...
 			}
+			k.l.Errorf("couldn't start monitoring of the kubernetes cluster: %s", err)
+			return nil, errors.Wrap(err, "couldn't start monitoring of the kubernetes cluster")
 		}
+	}
 
-		if clusterInfo.Operators == nil || clusterInfo.Operators.OlmOperatorVersion == "" {
-			operator := "victoriametrics-operator"
+	// Install the operators in the background.
+	go k.installDefaultOperators(operatorsToInstall)
```
Comment (gen1us2k) [inline managed/services/management/dbaas/kubernetes_server.go:370]: We may have a situation when we run StartMonitoring while VM operator is not installed, don't we?

**Review by gen1us2k — COMMENTED** [review]
That's weird that PR has changes from different PRs I suppose.

File: managed/services/management/dbaas/kubernetes_server.go (hunk 2 above)
Comment (percona-csalguero) [reply managed/services/management/dbaas/kubernetes_server.go:370]: You are right. Thanks.

**Review by gen1us2k — APPROVED** [review]

**Pull request closed without merging**